### DISCUSSION TITLE
Add option to cancel download while in beatmap listing

### DISCUSSION THREAD

**Discussion by Piggey** [ppy/osu#20930] [Ideas] [answered]
So I've noticed that once you start to download some map in beatmap listing section in osu!lazer there is no way to cancel the download there.

What makes it weird for me is that the cancellation of download is possible and already implemented, but it's in notifications.

I think it would be a really nice quality of live improvement to make it possible to cancel the download while in the beatmap listing section. Maybe the "download" button would change to "cancel" while it's downloading or something like that.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13055
